File: linear-datasructures/arrays/traverse/go/traverse1.go
package main

import "fmt"

func main() {
	//traversing
	v := []int{12, 34, 55, 67}
	for _, val := range v {
		println(val)
	}
	val := []int{34, 65, 86, 90, 21}
	next := append(val, 56)
	for i := 0; i < len(val); i++ {
		fmt.Println("Real value", val)
		fmt.Println("temporary value", next)
	}
}

File: linear-datasructures/arrays/traverse/python/traverse1.py
#traversing a data structure is to access or modify its elements, or to search 
#for a specific element or set of elements

arr = [12,34,56,77,88]

for i in range(len(arr)):
      print(arr[0:2])
      print(arr[i])
      
for a in range(len(arr)):
      u = arr.append(99) 
      print(u)     
      
new = [23,45,67,88,12]
new[2]= 99
print(new)  
# it will print out [23,45,99,88,12] but it will store in a temporary instance
# but wont be permanently modified , unless the initial array will be readjusted again
    